Formula/minimal-racket.rb

```diff
@@ -55,13 +59,20 @@ def install
       system "make"
       system "make", "install"
     end
+
+    inreplace racket_config, prefix, opt_prefix
   end
 
   def post_install
     # Run raco setup to make sure core libraries are properly compiled.
     # Sometimes the mtimes of .rkt and .zo files are messed up after a fresh
     # install, making Racket take 15s to start up because interpreting is slow.
-    system "#{bin}/raco", "setup"
+    system bin/"raco", "setup"
+
+    return unless racket_config.read.include?(prefix)
```

Comment:
This doesn't quite work the way I wanted it to, since this only fixes references to the current `prefix`, when we should be fixing references to the old one.